azure-pipelines.yml

```diff
@@ -0,0 +1,84 @@
+resources:
+  repositories:
+  - repository: self
+  - repository: das-platform-building-blocks
+    type: github
+    name: SkillsFundingAgency/das-platform-building-blocks
+    ref: refs/tags/0.3.1
+    endpoint: SFA
+  - repository: das-platform-automation
+    type: github
+    name: SkillsFundingAgency/das-platform-automation
+    ref: refs/tags/4.2.7
+    endpoint: SFA
+
+pr: none
+
+trigger:
+  batch: true
+  branches:
+    include:
+      - master
```

Comment:
* already has condition on PROD for master

Reply:
Changed to wildcard